<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserItem</class>
 <widget class="QWidget" name="UserItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>60</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="le_feeling">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>35</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lb_name">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string> name</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_icon">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>icon</string>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
